### PR TITLE
modified calculator.js

### PR DESCRIPTION
**Pull request by sansadev**

### CODE REVIEW

**Comment by MadEste** [comment]
Added a Pull Request.

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+let b = 3;
+let c = 2;
+let d = 3;
+
+if (a >= 6) {
+  console.log(a + 2);
+} else if (b <= 0) {
+  console.log(b / 5);
+}
+  else  {
```
Comment (MadEste) [inline calculator.js]: Try and be consistent with brackets. (move else up to line 10, this is only a coding style consideration)

**Comment by sansadev** [comment]
added a file called 'countown.js'. I created a for loop.

File: countdown.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+for(i=5; i>=0; i--)
+console.log(i);
```
Comment (MadEste) [inline countdown.js]: 👍

**Pull request closed without merging**